### PR TITLE
updates and rm run results in docs

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_iterable_source/issues/36
#37 

**This PR will result in the following new package version:**

v0.10.0
breaking change since this could affect runs and will result in downstream changes as well

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Breaking Change (`--full-refresh` required after upgrading)  
- Updates the materialization in `stg_iterable__user_history` from table to view in order to improve performance. A `--full-refresh` is required after upgrading to ensure there are no issues.

## Bug Fix 
- Removes the source freshness tests in the `src_iterable.yml`. This was originally causing run errors for users where tables weren't present in their schema but listed in the `src_iterable.yml`.
- Removes `created_at` from the uniqueness test for `stg_iterable__event`. Uniqueness is now tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id` (a Fivetran-created column derived from a hash of `user_id` and/or `email`).

## Documentation
- Updates the descriptions of timestamp-based fields. Previously they were described as milliseconds since epoch time, but they should be standard timestamps.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [na] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- This is just a removal of the source freshness tests in the src yml, allowing the models to run successfully in users environments. I also ran the downstream models with this branch, with no issue.

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: README.md
Comment (fivetran-joemarkiewicz) [inline README.md:1]: Can you be sure to update the "What does this package do" section to mention that this package does not provide source freshness checks. 

[Here](https://github.com/fivetran/dbt_aws_cloud_cost?tab=readme-ov-file#what-does-this-dbt-package-do) is how we documented this recently in the AWS Cloud Cost data model.
image

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-reneeli this looks good to me, but I have one documentation request to add before approval. Let me know if you have questions. Thanks!

File: README.md
Comment (fivetran-reneeli) [reply README.md:1]: thanks Joe, added that in. I also uncollapsed the Optional Steps section.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Just one comment to update the CHANGELOG. Otherwise looks good!

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md]: oh good point since this alters the schema-- updated

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:8]: Small request but we should list the breaking change about the bugfix.

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:9]: Also, there is no reference to the documentation changes. Please add that as an entry to a new documentation section.

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:11]: Please link and reference the PRs here instead of the Issues.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM just a few final requests to update the CHANGELOG

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md]: removed the links since we'll have them in the transforms.

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md:10]: updated

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md]: yes! added

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:10]: Request to make the breaking change the first entry in the CHANGELOG so it's the first thing users see.

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:15]: If there is no PR linked in the source for the contribution then we shouldn't include this section in the CHANGELOG. We should only include a contributing section when there is a PR linked within the package update.

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:6]: Great entry - suggestion to add a bit more context around the user_id field similar to what you shared with the event_id since these are both Fivetran-created and we should add more details around them.
```suggestion
- Removes `created_at` from the uniqueness test for `stg_iterable__event`. Uniqueness is tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id` (a Fivetran-created column derived from a hash of `user_id` and/or `email`).
```

**Review by fivetran-joemarkiewicz — APPROVED** [review]
This PR looks good, just a few final change requests before opening for release review.

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md:6]: ah good suggestion! thanks

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md:15]: Ah ok! That makes a lot of sense

File: CHANGELOG.md
Comment (fivetran-avinash) [inline CHANGELOG.md:7]:
```suggestion
## Bug Fixes 
```

File: CHANGELOG.md
Comment (fivetran-avinash) [inline CHANGELOG.md:11]:
```suggestion
## Documentation Update
```

File: CHANGELOG.md
Comment (fivetran-avinash) [inline CHANGELOG.md:9]:
```suggestion
- Removes `created_at` from the uniqueness test for `stg_iterable__event` in `stg_iterable.yml`. Uniqueness is now tested solely on `unique_event_id`, a surrogate key made up of `event_id` (`_fivetran_id` in the raw table, which is a Fivetran-created unique identifier derived from hashing campaign_id, created_at, and event_name) and `_fivetran_user_id` (a Fivetran-created column derived from a hash of `user_id` and/or `email`).
```

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-reneeli Great work! A few comments, but should be good to merge after that.

**Pull request merged**
